core/trino-main/src/main/java/io/trino/server/testing/TestingTrinoServer.java

```diff
@@ -379,6 +380,8 @@ private TestingTrinoServer(
 
         pluginInstaller = injector.getInstance(PluginInstaller.class);
 
+        plugins.forEach(pluginInstaller::installPlugin);
```

Comment:
You don't need that. You can add a setup action that will install a plugin. Remove